aiida/cmdline/commands/cmd_restapi.py

```diff
@@ -26,34 +26,44 @@
 
 
 @verdi.command('restapi')
-@click.option('-H', '--host', type=click.STRING, default='127.0.0.1', help='the hostname to use')
-@click.option('-p', '--port', type=click.INT, default=5000, help='the port to use')
+@click.option('-H', '--hostname', type=click.STRING, default='127.0.0.1', help='Hostname')
+@click.option('-P', '--port', type=click.INT, default=5000, help='Port number')
 @click.option(
     '-c',
     '--config-dir',
     type=click.Path(exists=True),
     default=DEFAULT_CONFIG_DIR,
     help='the path of the configuration directory')
-def restapi(host, port, config_dir):
+@click.option('--debug', 'debug', is_flag=True, default=False, help='run app in debug mode')
+@click.option(
+    '--wsgi-profile',
+    'wsgi_profile',
+    is_flag=True,
+    default=False,
+    help='to use WSGI profiler middleware for finding bottlenecks in web application')
+@click.option('--hookup/--no-hookup', 'hookup', is_flag=True, default=True, help='to hookup app')
+def restapi(hostname, port, config_dir, debug, wsgi_profile, hookup):
     """
     Run the AiiDA REST API server
 
     Example Usage:
 
         \b
-        verdi -p <profile_name> restapi --host 127.0.0.5 --port 6789 --config-dir <location of the config.py file>
+        verdi -p <profile_name> restapi --hostname 127.0.0.5 --port 6789 --config-dir <location of the config.py file>
+        --debug --wsgi-profile --hookup
     """
     from aiida.restapi.api import App, AiidaApi
     from aiida.restapi.run_api import run_api
 
     # Construct parameter dictionary
     kwargs = dict(
-        hookup=True,
         prog_name='verdi-restapi',
-        default_host=host,
-        default_port=port,
+        hostname=hostname,
+        port=port,
```

Comment:
the same goes for the `default_config` no? As in, this should just be `config`

Reply:
Corrected.